### PR TITLE
Search section

### PR DESCRIPTION
**Pull request by katbow**
for #451

 * Filters have had: Round edges, Remove background, Change text & border colour
* Add blurred image background instead of dark blue
* Centre search bar & make full width
* Centre filters on mobile, align left on desktop
* Make search bar transparent (unless text cannot be black this way, then keep white #455
* On focus, search bar outline is orange
* Change search button to new primary button design #448
* Move search button to half way point of the bottom of the section, below & in line with right of search bar.

Only need to add blue instead of white plus, but need to wait on asset. Good to be merged in now 👍

### CODE REVIEW

File: web/static/css/app.css [hunk 1]
```diff
@@ -344,6 +344,12 @@ footer p a {
   background-image: url("../images/whatsapp_hover.svg");
 }
 
+
+.blurred_background {
+  background-image: url("../images/blurred_gradient.png");
+  background-repeat: no-repeat;
```
Comment (Cleop) [inline web/static/css/app.css]: Alternative in tachyons is `bg-center` which centers the background but also sets no-repeat.

**Review by Cleop — CHANGES_REQUESTED** [review]
Looks great. The only thing that I think needs adding is the shadow on the selected state of the primary button: 
image

**Comment by katbow** [comment]
@Cleop what does the selected state mean? For the moment it's clicked only?

**Review by Cleop — APPROVED** [review]

**Comment by Cleop** [comment]
Thanks for adding that in @katbow 👍

**Pull request merged**